### PR TITLE
Cli

### PR DESCRIPTION
**Pull request by eggplants**
```shellsession
$ rqw -h
usage: rqw [-h] (-f FILE | -Q QUERY) [-F FORMAT] [-e URI] [-q] [-V]

sparqlwrapper CLI

optional arguments:
  -h, --help                  show this help message and exit
  -f FILE, --file FILE        query with sparql file (stdin: -) (default: None)
  -Q QUERY, --query QUERY     query with string (default: None)
  -F FORMAT, --format FORMAT  response format (default: json)
  -e URI, --endpoint URI      sparql endpoint (default: http://dbpedia.org/sparql)
  -q, --quiet                 supress warnings (default: False)
  -V, --version               show program's version number and exit

allowed formats:
  - json
  - xml
  - turtle
  - n3
  - rdf
  - rdf+xml
  - csv
  - tsv

$ rqw -Q 'SELECT ?s WHERE { ?s ?p ?o. } limit 1'
{
    "head": {
        "link": [],
        "vars": [
            "s"
        ]
    },
    "results": {
        "distinct": false,
        "ordered": true,
        "bindings": [
            {
                "s": {
                    "type": "uri",
                    "value": "http://www.openlinksw.com/virtrdf-data-formats#default-iid"
                }
            }
        ]
    }
}
```

### CODE REVIEW

**Comment by aucampia** [comment]
@eggplants looks great, but maybe this would be better in rdflib main? Not sure this is the right place for it.

**Comment by eggplants** [comment]
Do you think it is better `cli.py` than `main.py`?
I will rename this.

**Comment by aucampia** [comment]
No I mean it may be better to place this in https://github.com/RDFLib/rdflib/ than sparqlwrapper

**Comment by eggplants** [comment]
Oh, I understood.

**Comment by aucampia** [comment]
maybe @ashleysommer @nicholascar can also give some feedback here, but this is something which will be very useful in my opinion.

**Comment by aucampia** [comment]
Actually, I looked now, I see the CLI executes against remote endpoints, in which case this may be more appropriate, but we need something like this in RDFLib for files also. I will think about it a bit, the CI in this repo is a bit bitrotten, will look at it when I have time.

**Comment by eggplants** [comment]
Thank you so much. I think this project is maybe outdated but still very useful for me. I'll try to do implementing a CLI for rdflib, but once I do, I'd really like to see the CLI for sparqlwrapper merged.

**Comment by nicholascar** [comment]
I think this script should stay in SPARQLWrapper and not be put in RDFlib main. The projects really are quite distinct, at least in terms of maintenance!

A lot could be done to modernise SPARQLWrapper but it's still very useful as it is - it still works well for most SPARQL endpoints - but we will have SPARQL 1.2 coming soon and people will have to pay it serious attention then.

So, I think this is a great incremental contribution to SPARQLWrapper and, as per the general rules for contributions: if someone makes a contribution, it must be because it's important to them, so we will accept it as long as it doesn't break anything and this one doesn't, so it's msotyl good to go.

However, could you please add in any sort of tests? That would really allow us to ensure it's maintainable going forward. Thanks!

**Comment by eggplants** [comment]
𝙍𝙚𝙖𝙙𝙮 𝙛𝙤𝙧 𝙧𝙚𝙫𝙞𝙚𝙬…

**Review by nicholascar — APPROVED** [review]
@eggplants this is a great PR: nice new functionality, neat code and sensible tests!

We don't have  automated testing for SPARQLWrapper yet but clearly this PR doesn't touch existing tests - new functionality in new files only - so we can approve this without solving that automated testing problem for now.

If you would like to help with other aspects of SPARQLWrapper, especially updating the testing, that would be most appreciated.

Over to @aucampia for a second review.

**Review by aucampia — APPROVED** [review]
Apologeties for not reviewing earlier, code looks good, , the change looks great, maybe for posterity sake it would be good to note what `rwq` is an abbreviation for.

File: test/cli_test.py
Comment (aucampia) [inline test/cli_test.py:15]: This can be written more succinctly as:
```python
from pathlib import Path
_top_level_path = Path(__file__).parent.parent.absolute()
```
However what you did works, and I'm fine with it.

**Comment by nicholascar** [comment]
Yeah, good point, what does `rwq` stand for? I think I should be able to work it out but I haven't quite go there!

I might pick up some tips from this PR to improve the command line interface for [pyLODE](https://github.com/RDFLib/pyLODE)...

But I'll just merge this in now and if we work out what `rwq` stands for, I'll just merge in a note.

**Comment by eggplants** [comment]
`rqw` stands for:
- rq=spaRQl
- w=wrapper

**Pull request merged**
